docs/api-markdown-documenter/api-documentation-layout.js

```diff
@@ -18,25 +20,49 @@ import { AlertNode } from "./alert-node.js";
 const customExamplesSectionTitle = "Usage";
 const customThrowsSectionTitle = "Error Handling";
 
-const alphaWarning = SpanNode.createFromPlainText(
-	"WARNING: This API is provided as an alpha preview and may change without notice. Use at your own risk.",
-);
-const betaWarning = SpanNode.createFromPlainText(
-	"WARNING: This API is provided as a beta preview and may change without notice. Use at your own risk.",
-);
+// Temporary workaround for items tagged as `@alpha` (to mean "legacy").
+// This messaging should be changed back to standard "alpha" terminology once we have
+// cleaned up our tag meanings.
+function createAlphaWarning(apiItem) {
+	const packageName = apiItem.getAssociatedPackage().displayName;
+	return new AlertNode(
+		[
+			new SpanNode([
+				new PlainTextNode("To use, import via "),
+				CodeSpanNode.createFromPlainText(`${packageName}/legacy`),
+				new PlainTextNode("."),
+			]),
+		],
+		/* alertKind: */ "note",
```

Comment:
discrepancy here with this being 'note' while for beta warning its a 'warning'. Just pointing out if incase this is not intended.

Reply:
It is intentional. We make real support guarantees for our "legacy" APIs, and they _are_ intended for production use. But we don't make support guarantees for beta APIs. So I figured the distinction in alert level was merited.